source/ssb-chart/templates/access-ingress-external-istio-gateway-netpol.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if ((((.Values.access).ingress).external).gateways) }}
+{{/*  TODO: CHeck if this netpol already exist in platform or k8s-config-overlays */}}
+# Allow ingress from istio ingress gateway
```

Comment:
Do you think this Netpol is needed, @vidaks ? I could not find a policy specifying the same after a quick search in platform and k8s-config-overlays repo.
The reason I added this is that I also added a default deny all ingress netpol.

Reply:
When you now add a "default-deny" network policy, you will probably need this to allow traffic from the istio ingress gw. I suppose you need this netpol in addition to an AuthorizationPolicy with the same "from" selector and a VirtualService to allow external ingress.

Reply:
If I remember correctly this rule was covered by he (old) policy that allowed all traffic within the cluster.
`namespace: {}`.

Reply:
The namespace selector should probably also select the "istio-system" namespace or am i missing something?